README.md

```diff
@@ -58,6 +58,11 @@ visual_studio_code_build: stable
 # with a trailing slash.
 visual_studio_code_mirror: 'https://packages.microsoft.com'
 
+# skip task to add repo for remote package manager
+# if set to true, the task 'install VS Code repo (apt/yum/dnf/zypper)' will be skipped
```

Comment:
Please update this to use `yes`/`no` as below.

Reply:
i added the change into a separate commit.
i can squash it if you like

Reply:
Sorry, I must have started the review between these changes, all good now.